Doc/library/http.cookiejar.rst

```diff
@@ -633,6 +633,11 @@ internal consistency, so you should know what you're doing if you do that.
    case :attr:`version` is 0.
 
 
+.. attribute:: Cookie.domain
```

Comment:
I would place it between `port` and `path` -- this is the order in repr (ignoring derived attributes like `port_specified` etc).